lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -90,15 +90,11 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
 
         if (this.isAlfrescoAndLocal()) {
             const destinationFolderPath = this.getAliasAndRelativePathFromDestinationFolderPath(this.field.params.fileSource.destinationFolderPath);
+            const opts = { relativePath: destinationFolderPath.path };
 
-            if (destinationFolderPath.path) {
-                const opts = { relativePath: destinationFolderPath.path };
-                await this.contentNodeSelectorService.fetchNodeIdFromRelativePath(destinationFolderPath.alias, opts).then((nodeId: string) => {
-                    this.rootNodeId = nodeId ? nodeId : destinationFolderPath.alias;
-                });
-            } else {
-                this.rootNodeId = destinationFolderPath.alias;
-            }
+            await this.contentNodeSelectorService.fetchNodeIdFromRelativePath(destinationFolderPath.alias, opts).then((nodeId: string) => {
```

Comment:
are you mixing await with the Observable? it's not going to work in either cases. should be either `await` or `then`